### DISCUSSION TITLE
Slogan of envd

### DISCUSSION THREAD

**Discussion by gaocegege** [tensorchord/envd#554] [General]
Now we use `Development environment for ML` as the slogan. We did not discuss it further, thus I think we can have a better slogan with the help of the community :tada: 

BTW, the name `envd` comes from this issue #2 

Besides this, we also need a logo #326

**Comment by terrytangyuan** [comment]
Some ideas:
1. Build it once and deploy it everywhere.
1. Out-of-box data science workbench for everyone.
